### PR TITLE
Update README and include full MPL2 license file.

### PR DESCRIPTION
**Pull request by davehunt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,131 +1,106 @@
...
+
+1. Follow our simple [style guide][].
+2. Fork this project with your own GitHub account.
+3. Make sure all tests are passing, and submit a pull request.
+4. Always feel free to reach out to us and ask questions.
+
+[guide]: http://firefox-test-engineering.readthedocs.io/en/latest/guide/index.html
+[git clone]: https://help.github.com/articles/cloning-a-repository/
+[git fork]: https://help.github.com/articles/fork-a-repo/
+[staging]: https://mozillians.allizom.org/
```
Comment (stephendonner) [inline README.md]: This should be https://web-mozillians-staging.production.paas.mozilla.community (https://github.com/mozilla/mozillians-tests/blob/baacd928f72f06044137bcc5ea83b643a0db4fd2/tox.ini#L28)

Comment (davehunt) [reply README.md]: Oops! Thanks! :)

**Comment by davehunt** [comment]
@stephendonner URL updated - thanks for catching that. Mind re-reviewing/merging?

**Comment by stephendonner** [comment]
My pleasure; thanks again!

**Pull request merged**